scripts/restart_pipelines.py

```diff
@@ -2,19 +2,27 @@
 # future could read this out of ssm
 import boto3
 import json
+import time
 
 # state machine to restart
-statemachine_arn = "arn:aws:states:us-east-1:333680067100:stateMachine:manifest-pipeline-mets-test-SchemaStateMachine"
+statemachine_arn = "arn:aws:states:us-east-1:230391840102:stateMachine:marble-manifest-prod-SchemaStateMachine"
 # bucket to find process items
-bucket = "manifest-pipeline-mets-test-processbucket-uryjoy854u02"
+bucket = "marble-manifest-prod-processbucket-kskqchthxshg"
 
 client = boto3.client('stepfunctions')
 s3 = boto3.resource('s3')
 result = s3.meta.client.list_objects(Bucket=bucket, Prefix="process/", Delimiter='/')
+count = 1
 
 for o in result.get('CommonPrefixes'):
     id = o.get('Prefix').replace("process/", "").replace("/", "")
     print("Starting:", id)
+    if count % 5 == 0:
+        print("sleeping")
+        time.sleep(150)
+
+    count += 1
+
```

Comment:
why?

Reply:
![So-Tired](https://user-images.githubusercontent.com/416559/68154078-b86b9d80-ff14-11e9-9bda-da22354afa89.jpg)

Reply:
because you can only have so many ec2 tasks running at once and if i let it start all the pipelines it will fail on the image step. this was my fast way of getting it to work..